packages/synapse-react-client/src/components/HeaderCard/HeaderCardV2.tsx

```diff
@@ -196,122 +275,216 @@ function HeaderCardV2({
+              descriptionSubTitle=""
+              descriptionConfig={descriptionConfiguration}
+            />
+          </Box>
           {values && (
-            <Grid
-              item
-              xs={12}
-              md={forceStackedLayout ? 12 : 4}
+            <Box
+              ref={metadataRef}
               sx={{
-                borderLeft: {
-                  xs: 'none',
-                  md: forceStackedLayout
-                    ? 'none'
-                    : `1px solid ${theme.palette.divider}`,
-                },
-                pl: { xs: 0, md: forceStackedLayout ? 0 : 3 },
-                mt: { xs: 2, md: 0 },
+                width: useStackedLayout ? '100%' : 'auto',
+                marginTop: useStackedLayout ? 2 : 0,
               }}
             >
-              <CardFooter
-                isHeader={true}
-                secondaryLabelLimit={secondaryLabelLimit}
-                values={values}
-              />
-            </Grid>
+              <MetadataTable data={values} />
+            </Box>
           )}
-        </Grid>
+        </Box>
+
+        {ctaButtons && ctaButtons.length > 0 && (
+          <Stack
+            direction={{ xs: 'column', sm: 'row' }}
+            spacing={2}
+            sx={{ mt: 2 }}
+          >
+            {ctaButtons.map((buttonProps, index) => (
+              <Button
+                key={index}
+                variant={buttonProps.variant || 'contained'}
+                href={buttonProps.href}
+                sx={{
+                  width: { xs: '100%', sm: 'auto' },
+                  ...(buttonProps.sx || {}),
+                }}
+                endIcon={buttonProps.endIcon}
+              >
+                {buttonProps.label}
+              </Button>
+            ))}
+          </Stack>
+        )}
       </Box>
     </Card>
   )
 }
 
+type MetadataTableProps = {
+  data: string[][]
+}
+
+function MetadataTable({ data }: MetadataTableProps) {
+  return (
+    <table
+      style={{
+        width: '100%',
+        borderCollapse: 'collapse',
+      }}
+    >
+      <tbody>
+        {data.map((item, index) => (
+          <tr key={item[2] || index}>
+            <td
+              style={{
+                padding: '0px 16px 8px 0',
+                verticalAlign: 'top',
+                whiteSpace: 'nowrap',
+                fontWeight: 'bold',
+              }}
+            >
+              {item[0]}
+            </td>
+            <td
+              style={{
+                paddingBottom: '12px',
+                verticalAlign: 'top',
+                wordBreak: 'break-word',
+              }}
+            >
+              {item[1]}
+            </td>
+          </tr>
+        ))}
+      </tbody>
+    </table>
+  )
+}
+
 export default HeaderCardV2
+
+// Remove when i have storyboard working again or real data to test with
+import { LaunchOutlined } from '@mui/icons-material'
+const ctaButtonsForTesting: CTAButton[] = [
```

Comment:
Were you able to get storyboard working again?  In any case, this should probably be removed before merge